project/src/js/i18n.js

```diff
@@ -23,6 +23,7 @@ var untranslated = {
   allowAnalytics: 'Allow anonymous statistics',
   shareGame: 'Share game',
   sharePGN: 'Share PGN',
+  onlineFriends: 'Online friends',
```

Comment:
you don't want to add this: the `onlineFriends` key already exist and is translated. Here is only for untranslated keys.